test/google_lookup_test.rb

```diff
@@ -39,6 +39,33 @@ def test_google_request_denied_read_lat_lon_coordinates
     end
   end
 
+  def test_url_non_enterprise
+    result = lookup.send(:url, '123', '123').split('&')
+
+    assert_equal '/maps/api/timezone/json?location=123%2C123', result[0]
+    assert result[1].start_with?('timestamp=')
+    assert_equal 'key=123abc', result[2]
+    assert_equal 3, result.count
+  end
+
+  def test_url_enterprise
+    Timezone::Configure.begin do |c|
+      c.google_client = '123&asdf'
+    end
+    result = lookup.send(:url, '123', '123').split('&')
+
+    assert_equal '/maps/api/timezone/json?location=123%2C123', result[0]
+    assert result[1].start_with?('timestamp=')
+    assert_equal 'client=123%26asdf', result[2]
+    assert result[3].start_with?('signature=')
```

Comment:
Would be nice to have a signature validation here. Check out using `timecop` to validate that you are properly computing the signature.